mass_mailing_custom_unsubscribe/__openerp__.py

```diff
@@ -21,13 +21,15 @@
 #
 ##############################################################################
 {
-    'name': "Customizable unsubscribe link on mass mailing emails",
+    'name': "Customizable unsubscription process on mass mailing emails",
     'category': 'Marketing',
-    'version': '8.0.1.0.0',
+    'version': '8.0.2.0.0',
```

Comment:
This signal that the changes are significant enough so that migration work might be needed.
You probably want `8.0.1.1.0`, in case a module upgrade is enough for existing installations.